generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java

```diff
@@ -126,7 +127,8 @@ private DomContent person(JavaPerson person) {
 				td(person.name()), //
 				td(handleWithLink(person, JavaPerson::fediverse, JavaPerson::fediverseLink)), //
 				td(handleWithLink(person, JavaPerson::twitter, JavaPerson::twitterLink)), //
-				td(handleWithLink(person, JavaPerson::github, JavaPerson::githubLink)));
+				td(handleWithLink(person, JavaPerson::github, JavaPerson::githubLink)), //
+				td(handleWithLink(person, JavaPerson::github, JavaPerson::redditLink)));
```

Comment:
```suggestion
				td(handleWithLink(person, JavaPerson::reddit, JavaPerson::redditLink)));
```
reddit here too, I think